### PR TITLE
OHRM5X-1123: Config store and i18n support added

### PR DESCRIPTION
**Pull request by Super-Chama**

### CODE REVIEW

File: storybook/package.json [hunk 1]
```diff
@@ -8,14 +8,14 @@
...
     "@babel/core": "^7.12.10",
-    "@storybook/addon-a11y": "^6.3.0",
-    "@storybook/addon-actions": "^6.3.0",
-    "@storybook/addon-essentials": "^6.3.0",
-    "@storybook/addon-links": "^6.3.0",
-    "@storybook/addon-viewport": "^6.3.0",
-    "@storybook/addons": "^6.3.0",
-    "@storybook/theming": "^6.3.0",
-    "@storybook/vue3": "^6.3.0",
+    "@storybook/addon-a11y": "^6.4.18",
```
Comment (RajithaKumara) [inline storybook/package.json:11]: :smiley:

**Review by RajithaKumara — APPROVED** [review]

**Comment by RajithaKumara** [comment]
@samanthajayasinghe please review when you have time

File: components/package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "core-js": "^3.6.5",
     "date-fns": "^2.22.1",
     "mitt": "^2.1.0",
-    "nanoid": "^3.1.21",
+    "nanoid": "^3.1.31",
```
Comment (RajithaKumara) [inline components/package.json:16]: Can't we go for `3.2.0`?

Comment (Super-Chama) [reply components/package.json:16]: added in #124

**Pull request merged**